src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -233,6 +235,29 @@ public void queueDelete(Location l, boolean destroy) {
         deletionQueue.put(l, destroy);
     }
 
+
+    @ParametersAreNonnullByDefault
+    public void queueDelete(Collection<Location> locations, boolean destroy) {
+        Validate.notNull(locations, "Locations must not be null");
+
+        Map<Location, Boolean> toDelete = new HashMap<>(locations.size(), 1.0F);
+        for (Location location : locations) {
+            Validate.notNull(location, "Locations must not contain null locations");
+            toDelete.put(location, destroy);
+        }
+        deletionQueue.putAll(toDelete);
+    }
+
+    @ParametersAreNonnullByDefault
+    public void queueDelete(Map<Location, Boolean> locations) {
```

Comment:
Why the boxed bool?

Reply:
Is it possible to have maps with primitives?

Reply:
![image](https://github.com/Slimefun/Slimefun4/assets/65748158/f29d6777-199d-4e91-af55-8b8e685cc4ba)
Yeah like I thought you can't do that, or at least not with the standard `Map` and `boolean`

Reply:
... I haven't wrote Java in so long :NotLikeThis:

It is but we'd need to use like fastutil

Reply:
Gotcha